src/main/java/mate/zorii/bookstore/model/ShoppingCart.java

```diff
@@ -0,0 +1,31 @@
+package mate.zorii.bookstore.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "shoppingCarts")
+@Getter
+@Setter
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne(fetch = FetchType.EAGER)
```

Comment:
```suggestion
    @MapsId
    @OneToOne(fetch = FetchType.LAZY)
```

you can use MapsId here